src/sentry/models/relay.py

```diff
@@ -17,6 +17,7 @@ class RelayUsage(Model):
     version = models.CharField(max_length=32, default="0.0.1")
     first_seen = models.DateTimeField(default=timezone.now)
     last_seen = models.DateTimeField(default=timezone.now)
+    public_key = models.CharField(max_length=200, default="", db_index=True)
```

Comment:
We typically recommend that you just use a `models.TextField` here. This just helps you avoid a painful migration if you decide to change the column length at some point in the future. If you want to enforce a length in Django, @mattrobenolt wrote an alternative `CharField` that is backed by a `text` column in postgres: https://github.com/getsentry/sentry/blob/3b5490df3b37ce20a3d01ae3b7c1114ee1e00c3f/src/sentry/db/models/fields/text.py#L28

Reply:
@wedamija We'd probably want to keep this consistent with the other model in that file that uses a CharField for `public_key`, too. WDYT?

Reply:
So I did a bit of reading of this, and it might actually no longer be a problem to change the lengths of varchar cols: https://stackoverflow.com/questions/48693373/does-alter-column-type-varcharn-rewrite-the-table-in-postgres-9-6.

Really up to y'all about how likely you are to need to change the length of this column. IMO, there's not a lot of reason to use a varchar in most cases, but I'm not going to block on it or anything.

Reply:
Ok, thanks. My understanding is that for a while VARCHAR has effectively been implemented as TEXT in pg, but I'd have to go back and check. The only reason for me to keep it as CharField now is consistency with other models that have the same `public_key`.

Reply:
Yeah, you're right that it's a `TEXT` column under the hood. It used to be the case that changing the length would cause a table rewrite, which is not great in production. This may have changed though.